site/_docs/history.md

```diff
@@ -29,6 +29,162 @@ Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
 
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.30.0">1.30.0</a> / 2022-03-04
+{: #v1-30-0}
+
+This release comes two months after [1.29.0](#v1-29-0),
+contains contributions from 29 authors,
+and resolves 37 issues.
+
+Among others, it is worth highlighting the following.
+
+* [Babel parser support MySQL NULL-safe equal operator '<=>'](https://issues.apache.org/jira/browse/CALCITE-4980)
+* [Support SQL hints for temporal table join](https://issues.apache.org/jira/browse/CALCITE-4967)
+* [Fluent test fixtures so that dependent projects can write parser, validator and rules tests](https://issues.apache.org/jira/browse/CALCITE-4885">CALCITE-4885)
+* [Add functions `ARRAY_CONCAT`, `ARRAY_REVERSE`, `ARRAY_LENGTH` for BigQuery dialect](https://issues.apache.org/jira/browse/CALCITE-4822">CALCITE-4822)
+* [Vulnerability issues CVE-2021-27568 fixed](https://nvd.nist.gov/vuln/detail/CVE-2021-27568)
```

Comment:
minor: "issues" should be "issue"

Reply:
Revised. Thanks for your careful review.